k8s/prod/common/xui/approve-org.yaml

```
@@ -22,6 +22,9 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/xui/ao-webapp:prod-5a4826e7
       ingressHost: administer-orgs.platform.hmcts.net
+      environment:
+        IDAM_API_SERVICE: https://idam-api.platform.hmcts.net
+        IDAM_WEB_SERVICE: https://idam-web-public.platform.hmcts.net
```

https://github.com/hmcts/rpx-xui-manage-organisations/blob/develop/infrastructure/prod.tfvars#L6
```suggestion
        IDAM_WEB_SERVICE: https://hmcts-access.service.gov.uk
```